controllers/translateController.js

```diff
@@ -21,9 +24,22 @@ const translateController = async (text, sourceLanguage, nameOfTranslation, opti
 
     language.files.forEach(fileName => {
       const { file: languageJson, parsedPath } = getOrCreateJsonFile(basePath, fileName);
-      setDeepValue(languageJson, nameOfTranslation, result);
+      const existProperty = has(languageJson, nameOfTranslation); 
```

Comment:
I think both are valids , but like more the existProperty one since as it starts with "exists" is more clear that it is a Boolean, it is not?

Reply:
what about haveProperty?

Reply:
I think I am confused about what name are you suggesting to change, I though you mean the variable existProperty, but now I am not sure hehe

Can you please clarify if you mean to change the name to the `has` function or to change the name to the `existsProperty` variable please?

Reply:
Yeah, the current `existProperty` sounds weird to me, in Spanish "Propiedad existe", notice the meaning is the same but the words are switched.

Reply:
I think `has` function can have another name as `hasProperty`, which sounds good for this function because is more descriptive and it's still a short name.

About the variable `existProperty` I think the name is enough clear, not sure if it is breaking any english grammatic rule or something like that is making it confuse, in that case I could change the name for `isPropertyInUse` or something like that

What do you think?

Reply:
Indeed, is some grammar thing hehe

Reply:
> Yeah, the current `existProperty` sounds weird to me, in Spanish "Propiedad existe", notice the meaning is the same but the words are switched.

Oh got it. Thanks for the feedback!. In that case I'll will check for alternatives

Reply:
Added some new variable names here, please take a look when u can! https://github.com/victor-heliomar/i18n-translation-generator/pull/18/commits/6d0a760168cb1164753dd774055126754a5c1c41